actpool/actpool.go

```diff
@@ -23,6 +24,32 @@ import (
 	"github.com/iotexproject/iotex-core/pkg/log"
 )
 
+var (
+	actpoolMtc = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "iotex_actpool_rejection_metrics",
+		Help: "actpool metrics.",
+	}, []string{"type"})
+	invalidCallerPk        = "invalidCallerPk"
```

Comment:
Maybe it's more straightforward to use string const in the code bellow directly.

Reply:
ok,I'll change it